docker/tests/containerTestCommon.psm1

```diff
@@ -213,6 +213,56 @@ function Get-ContainerPowerShellVersion
     return (Get-Content -Encoding Ascii $testContext.resolvedLogPath)[0]
 }
 
+
+function Get-DefaultPreviewConfigForPackageValidation
+{
+    @{  'centos7'='rhel.7';
+        'debian.8'='debian.8';
+        'debian.9'='debian.9';
+        'fedora26'='rhel.7';
+        'fedora27'='rhel.7';
+        'fedora28'='rhel.7';
+        'opensuse42.2'='linux-x64.tar.gz';
+        'opensuse42.3'='linux-x64.tar.gz';
+        'ubuntu14.04'='ubuntu.14.04';
+        'ubuntu16.04'='ubuntu.16.04';
+        'ubuntu18.04'='ubuntu.18.04'
+    }
+}
+
+function Get-DefaultStableConfigForPackageValidation
+{
+    @{  'centos7'='rhel.7';
+        'debian.8'='debian.8';
+        'debian.9'='debian.9';
+        'fedora26'='rhel.7';
+        'fedora27'='rhel.7';
+        'opensuse42.2'='linux-x64.tar.gz';
+        'opensuse42.3'='linux-x64.tar.gz';
+        'ubuntu14.04'='ubuntu.14.04';
+        'ubuntu16.04'='ubuntu.16.04'
+    }
+}
+
+# Returns a list of files in a specified Azure container.
+function Get-PackageNamesOnAzureBlob
+{
+    param(
+        [string]
+        $ContainerUrl,
+        
+        # $SAS (shared access signature) param should include beginning '?' and trailing '&'
+        [string]
+        $SAS
+    )
+
+
+    $responce = Invoke-RestMethod -Method Get -Uri $($ContainerUrl + $SAS + 'restype=container&comp=list')
+
+    $xmlResponce = [xml]$responce.Remove(0,3) # remove some bad chars in the beginning that break XML parsing
```

Comment:
This is fragile. It needs to remove chars based on where it finds beginning of XML code rather than removing hardcoded number of chars.

Reply:
Fixed.